docs/book/src/reference/providers.md

```diff
@@ -0,0 +1,24 @@
+## Provider Implementations
```

Comment:
We've seemed to have lost any discussion of API adoption, I think that should be preserved somehow.

Reply:
We can port material as we go, this is just to have a starting point

Reply:
Right, but I want to avoid any erasure of the existing community as we do.

Reply:
Can you point to the content you’d like to port as part of this PR?

Reply:
https://github.com/kubernetes-sigs/cluster-api/blob/13ecbf0897cf6ecb8ca0eb24b0fbc53a00855270/README.md#api-adoption

Reply:
Added, thanks